miden-tx/tests/integration/main.rs

```diff
@@ -10,135 +10,24 @@ use miden_objects::{
-    pub notes: Vec<InputNote>,
-    pub tx_args: TransactionArgs,
-}
-
-impl MockDataStore {
-    pub fn new() -> Self {
-        let (tx_inputs, tx_args) =
-            mock_inputs(MockAccountType::StandardExisting, AssetPreservationStatus::Preserved);
-        let (account, _, block_header, block_chain, notes) = tx_inputs.into_parts();
-        Self {
-            account,
-            block_header,
-            block_chain,
-            notes: notes.into_vec(),
-            tx_args,
-        }
-    }
-
-    pub fn with_existing(account: Option<Account>, input_notes: Option<Vec<Note>>) -> Self {
-        let (
-            account,
-            block_header,
-            block_chain,
-            consumed_notes,
-            _auxiliary_data_inputs,
-            created_notes,
-        ) = mock_inputs_with_existing(
-            MockAccountType::StandardExisting,
-            AssetPreservationStatus::Preserved,
-            account,
-            input_notes,
-        );
-        let output_notes = created_notes.into_iter().filter_map(|note| match note {
-            OutputNote::Full(note) => Some(note),
-            OutputNote::Partial(_) => None,
-            OutputNote::Header(_) => None,
-        });
-        let mut tx_args = TransactionArgs::default();
-        tx_args.extend_expected_output_notes(output_notes);
-
-        Self {
-            account,
-            block_header,
-            block_chain,
-            notes: consumed_notes,
-            tx_args,
-        }
-    }
-}
-
-impl Default for MockDataStore {
-    fn default() -> Self {
-        Self::new()
-    }
-}
-
-impl DataStore for MockDataStore {
-    fn get_transaction_inputs(
-        &self,
-        account_id: AccountId,
-        block_num: u32,
-        notes: &[NoteId],
-    ) -> Result<TransactionInputs, DataStoreError> {
-        assert_eq!(account_id, self.account.id());
-        assert_eq!(block_num, self.block_header.block_num());
-        assert_eq!(notes.len(), self.notes.len());
-
-        let notes = self
-            .notes
-            .iter()
-            .filter(|note| notes.contains(&note.id()))
-            .cloned()
-            .collect::<Vec<_>>();
-
-        Ok(TransactionInputs::new(
-            self.account.clone(),
-            None,
-            self.block_header,
-            self.block_chain.clone(),
-            InputNotes::new(notes).unwrap(),
-        )
-        .unwrap())
-    }
-
-    fn get_account_code(&self, account_id: AccountId) -> Result<ModuleAst, DataStoreError> {
-        assert_eq!(account_id, self.account.id());
-        Ok(self.account.code().module().clone())
-    }
-}
-
 // HELPER FUNCTIONS
 // ================================================================================================
 
 #[cfg(test)]
 pub fn prove_and_verify_transaction(
     executed_transaction: ExecutedTransaction,
 ) -> Result<(), TransactionVerifierError> {
+    use miden_objects::testing::constants::MIN_PROOF_SECURITY_LEVEL;
```

Comment:
Why is this coming from `testing` code? Wouldn't `MIN_PROOF_SECURITY_LEVEL` be the same for testing and non-testing code?

Reply:
The mock crate had its own constant and I had left it like that just in case (even though they were the same value), but let me remove the testing one.